client/components/mma/cancel/cancellationSaves/supporterplus/SupporterPlusOfferConfirmed.tsx

```diff
@@ -181,6 +186,16 @@ export const SupporterPlusOfferConfirmed = () => {
 		'yyyy-MM-dd',
 	).dateStr(DATE_FNS_LONG_OUTPUT_FORMAT);
 
+	const nextNonDiscountedPrice = routerState.nonDiscountedPayments.reduce(
+		(prev, current) =>
+			prev && prev.amount > current.amount ? prev : current,
+	).amount;
```

Comment:
Would it be worth extracting this to a method, since it's the same logic repeated in `SupporterPlusOffer.tsx`?